style_editor.html

```diff
@@ -8099,6 +8101,13 @@
   button.style.backgroundColor = framesPerUpdate == 60 ? "lightblue" : "";
 }
 
+function ClickStill() {
+  STATE_STILL = !STATE_STILL;
+  IN_FRAME = !IN_FRAME;
```

Comment:
This seems really weird.
a) how can we be inside the frame if we just clicked this button?
b) why are we suddenly inside the frame afterwards?

Reply:
Admittedly, this is just a quick dirty way to just have the motion and background swing sin stop.
It does not have anything to do with actually being in the frame, it's just to take on the behavior as if we *were* inside the frame.

Reply:
Ok I did it the right way instead.